src/@utils/aquarius.ts

```diff
@@ -47,6 +47,7 @@ export function generateBaseQuery(
           ...(baseQueryParams.filters || []),
           getFilterTerm('chainId', baseQueryParams.chainIds),
           getFilterTerm('_index', 'aquarius'),
+          getFilterTerm('accessDetails.type', ['free', 'fixed']),
```

Comment:
`accessDetails` is not on the ddo, we accidentally added it for some assets in the edit form. So doesn't have any purpose here

Reply:
🤔 mmm... this is a problem then. I'm looking at the [DDO docs](https://docs.oceanprotocol.com/concepts/did-ddo/#full-enhanced-ddo-example) now, but I'm not sure how to structure the query to get only "fixed" and "free" assets. What do you think is the best approach to filter these results?

Reply:
We don't need to filter these assets. Instead, we just need to alert the user on the Button. Let's just show "price not available" on these assets.